.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: end-of-file-fixer
+        exclude: ^images/.*/patches/.*\.patch$
+      - id: trailing-whitespace
+        exclude: ^images/.*/patches/.*\.patch$
+
+  - repo: https://github.com/psf/black
+    rev: 24.4.0
+    hooks:
+      - id: black
+
+  - repo: https://github.com/pycqa/flake8
+    rev: 7.0.0
+    hooks:
+      - id: flake8
+        args: [--max-line-length=100]
```

Comment:
you can remove this and replace with `.flake8` file with the config, see other projects we use :)

Reply:
actually, I found you already put a .flake8 file in project, but somehow pre-commit command ignore settings in file

Reply:
because I define pep8 with flake8 in tox.ini :)